eng/pipelines/ci.yml

```diff
@@ -71,9 +71,21 @@ parameters:
   type: boolean
   default: true
 
-extends:
-  template: templates/pipeline.yml
+resources:
+  pipelines:
+  - pipeline: DartLab
+    source: DartLab
+    branch: main
+  repositories:
+  - repository: DartLabTemplates
+    type: git
+    name: DartLab.Templates
+    ref: refs/heads/main
+
+stages:
+- template: templates/pipeline.yml
   parameters:
+    isOfficialBuild: false
```

Comment:
Is it intentional to set the official build as false here?

Reply:
Yes, see https://github.com/NuGet/NuGet.Client/pull/5632#discussion_r1490558324